scripts/emissions/bveCVX_special_emissions.py

```diff
@@ -0,0 +1,29 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+DEVELOPMENT = False
+
+
+
+# UPDATE THESE VARIABLES WITH THE NEEDS FOR A NEW GUARD LAUNCH OR SIMILAR SCENARIO
+STARTING_TIME = 1643241600
+ENDING_TIME = 1644534000
+AMOUNT_TO_EMIT_DURING_PERIOD = 100
+DURATION = ENDING_TIME - STARTING_TIME
+
+
+def main():
+    safe = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+    safe.init_badger()
+    rewards_logger = registry.eth.rewardsLogger
```

Comment:
think this should be set from techops, no?

also `rewards_logger` here is just an address string, still needs to be initiated as a contract object (`safe.contract(...)` is fastest for now).

Reply:
correct ser.